front/app/translations/en.json

```diff
@@ -13,6 +13,20 @@
   "app.components.AssignBudgetControl.remove": "Remove",
   "app.components.AssignBudgetControl.removeFromMyBasket": "Remove from my basket",
   "app.components.AssignBudgetControl.verifyAccountLinkText": "verify your account",
+  "app.components.AuthProviders.continue": "Continue",
```

Comment:
Thought we wanted to leave the message keys for now?

Reply:
I already changed these before we had the conversation. Should not be a problem as per what Brent explained. If we are missing the translations - I will change them back as to not block the release.

Reply:
You can quickly find out by opening a branch on Crowdin

Reply:
Alright, will check before merging!

Reply:
Translations look good.